deployment/vm/manager.go

```diff
@@ -154,3 +166,10 @@ func (m *manager) createAndRecordVM(agentID string, stemcell bistemcell.CloudSte
 
 	return cid, nil
 }
+
+func (m *manager) setVMNetworkInterfaces(vmNetworkInterfaces map[string]biproperty.Map, currentIP string, networkInterfaces map[string]biproperty.Map) {
+	for networkName, networkInterface := range networkInterfaces {
+		networkInterface["ip"] = currentIP
```

Comment:
The IP address will be only used for dynamic network, as Softlayer doesn’t support VIP network currently. This will be handled in [bosh-softlayer-cpi create_vm](https://github.com/cloudfoundry/bosh-softlayer-cpi-release/blob/master/src/bosh-softlayer-cpi/softlayer/vm/softlayer_virtual_guest_creator.go#L44).